packages/trader/src/App/Components/Elements/VideoStream/video-controls.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import VolumeControl from './volume-control';
+import PlaybackRateControl from './playback-rate-control';
+
+type TVideoControls = {
+    block_controls?: boolean;
+    current_time?: number;
+    is_playing?: boolean;
+    is_mobile?: boolean;
+    onRewind: (e: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+    dragStartHandler: (e: React.MouseEvent<HTMLElement, MouseEvent> | React.TouchEvent<HTMLElement>) => void;
+    onVolumeChange: (new_value: number) => void;
+    onPlaybackRateChange: (new_value: number) => void;
+    togglePlaying: (e: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+    video_duration?: number;
+    show_controls?: boolean;
+    volume?: number;
+    toggleMute: (new_value: boolean) => void;
+};
+
+const VideoControls = React.forwardRef<HTMLDivElement, TVideoControls>(
+    (
+        {
+            block_controls,
+            current_time,
+            is_playing,
+            is_mobile,
+            onRewind,
+            dragStartHandler,
+            onVolumeChange,
+            onPlaybackRateChange,
+            togglePlaying,
+            toggleMute,
+            show_controls,
+            video_duration,
+            volume,
+        },
+        ref
+    ) => {
+        const [is_drag_dot_visible, setIsDragDotVisible] = React.useState(false);
+
+        const formatTime = (time?: number) => {
+            if (time && !isNaN(time)) {
+                const minutes = Math.floor(time / 60);
+                const formatMinutes = minutes < 10 ? `0${minutes}` : `${minutes}`;
+                const seconds = Math.floor(time % 60);
+                const formatSeconds = seconds < 10 ? `0${seconds}` : `${seconds}`;
+                return `${formatMinutes}:${formatSeconds}`;
+            }
+
+            return '00:00';
+        };
+
+        return (
+            <div
+                className='player__controls__wrapper'
+                style={{
+                    opacity: `${show_controls ? '1' : '0'}`,
```

Comment:
let's use **classNames** instead of inline styles to set the required styles based on the `show_controls` flag :)